cmd/bucket-notification-utils_test.go

```diff
@@ -337,11 +332,6 @@ func TestUnmarshalSQSARN(t *testing.T) {
 			queueARN: "",
 			Type:     "",
 		},
-		// Invalid region 'us-west-1' in queue arn.
-		{
-			queueARN: "arn:minio:sqs:us-west-1:1:redis",
```

Comment:
@harshavardhana previously in tests server would run with region `us-east-1` not it runs with empty region. Because of this change:

```
--- a/cmd/globals.go
+++ b/cmd/globals.go
@@ -29,7 +29,7 @@ import (
 const (
        globalMinioCertExpireWarnDays = time.Hour * 24 * 30 // 30 days.
 
-       globalMinioDefaultRegion       = "us-east-1"
+       globalMinioDefaultRegion       = ""
        globalMinioDefaultOwnerID      = "minio"
        globalMinioDefaultStorageClass = "STANDARD"
        globalWindowsOSName            = "windows"
```

This is also causing the code coverage problem that you commented about.

We should have tests where server runs in non-empty region and clients fail when it does not use this region. I can fix the tests TestQueueARN and TestUnmarshalSQSARN to take care of the coverage issue. But we'll need some rework in test infra where it tests when server runs in non-empty region and clients signs using a different region